app/models/incremental_refresh_upload_task.rb

```diff
@@ -8,12 +8,14 @@ def post_upload_task
   def dispatch
     super
 
-    source.update!(:refresh_started_at   => Time.current,
-                   :refresh_finished_at  => nil,
-                   :refresh_task_id      => id,
-                   :last_refresh_message => "Message sent to RHC #{controller_message_id}",
-                   :refresh_state        => "Uploading")
+    if status == 'ok'
```

Comment:
@hsong-rh If the status is error where do we update the Source object with the error message since the UI only shows the data in the Source object

Reply:
Will update `source` when status is error: https://github.com/RedHatInsights/catalog_inventory-api/blob/master/app/services/post_upload_task_service.rb#L34